src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,7 +51,14 @@ private void configureButtonBindings() {
     driverStick = new Joystick(0);
     controlStick = new Joystick(1);
 
-    driveCommand = new ArcadeDriveWithJoystick(driverStick, Config.LEFT_CONTROL_STICK_Y, Config.INVERT_FORWARD, Config.RIGHT_CONTROL_STICK_X, Config.INVERT_SIDE);
+    /**
+     * Select different driveCommands to control the robot in different modes
+     */
+
+    //driveCommand = new ArcadeDriveWithJoystick(driverStick, Config.LEFT_CONTROL_STICK_Y, Config.INVERT_FIRST_AXIS, Config.RIGHT_CONTROL_STICK_X, Config.INVERT_SECOND_AXIS);
```

Comment:
You should remove the commented out code if it serves no purpose

Reply:
Wouldn't it be convenient to leave the commented code it since we can just uncomment the mode we want to use and comment out the one we don't so we can do quick changes?

Reply:
Convenient maybe but kinda messy. Part of what we're doing here is trying to emulate real software engineering best practices. Once we know what the drive team wants, this command won't really change, so need to keep other versions on "standby".

Reply:
Alright